Description: Removes <= 2.3.5 from lxml pip-requires
 Warn upstream on IRC about this, they said it's no problem to remove the
 version thing, and they will remove it upstream.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2013-05-16

--- heat-2013.1.orig/tools/pip-requires
+++ heat-2013.1/tools/pip-requires
@@ -17,7 +17,7 @@ httplib2
 iso8601>=0.1.4
 kombu>=1.0.4
 argparse
-lxml>=2.3,<=2.3.5
+lxml>=2.3
 sqlalchemy-migrate>=0.7.2
 python-novaclient
 PasteDeploy==1.5.0
